app/helpers/application_helper.rb

```diff
@@ -1,4 +1,14 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def make_this_a_list(options: {})
+    options[:document] # the original document
+    options[:field] # the field to render
+    options[:value] # the value of the field
+
+    list = +'<ul>'
+    options[:value].each { |identifier| list << "<li>#{identifier}</li>" }
+    list << '</ul>'
+    list
```

Comment:
Did you test this?  Does this work without it being marked `.html_safe`?

Reply:
Yes, it does work without being marked `.html_safe`.
![screen shot 2018-11-09 at 10 46 40 am](https://user-images.githubusercontent.com/588335/48272527-d3442880-e40c-11e8-9c92-472bdbd04c04.png)